hana/enable_cost_optimized.sls

```diff
@@ -46,10 +46,25 @@ install_hana_python_packages:
       - user: {{ node.sid.lower() }}adm
       - group: sapsys
       - enforce_toplevel: False
-      - source: {{ grains['hana_inst_folder']~'/DATA_UNITS/HDB_CLIENT_LINUX_X86_64/client/PYDBAPI.TGZ' }}
+      - source: {{ node.install.software_path~'/DATA_UNITS/HDB_CLIENT_LINUX_X86_64/client/PYDBAPI.TGZ' }}
+      - options: --strip=1 --wildcards 'hdbcli/*.py'
       - require:
         - reduce_memory_resources_{{ node.host+node.sid }}
         - setup_srHook_directory
+
+configure_ha_dr_provider_srTakeover:
+    file.append:
+      - name:  /hana/shared/{{ node.sid.upper() }}/global/hdb/custom/config/global.ini
+      - text: |
```

Comment:
I would add a new line before the text, otherwise th output is quite ugly

```
mode = sync
actual_mode = sync
site_name = SECONDARY_SITE_NAME
operation_mode = logreplay
[ha_dr_provider_srTakeover]
provider = srTakeover
path = /hana/shared/srHook
execution_order = 1
```
Not really visual